### DISCUSSION TITLE
5-3. 내가 더 빛날 무대가 다가오게 하는 방법 - 기록 편

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#17] [5장 다른 무대에도 서보고 싶어]
# 5-3. 내가 더 빛날 무대가 다가오게 하는 방법 - 기록 편 
현재 몸답고 있는 환경을 변화시키기 위해 여러모로 노력해봤으나 별로 변한 점이 없다면 성장을 위해 다른 무대로 이동하는 것이 좋은 선택일 수 있습니다. 이때 내게 맞는 무대를 찾는 방법은 그 무대가 내게 다가오도록 만드는 것입니다. 보통은 내게 맞는 무대를 전략적으로 찾은 후 그에 맞춰 준비하는 경우가 대부분입니다. 하지만 평소에 나의 경험을 잘 기록하고 드러냈다면 자연스럽게 다음 무대를 위한 기회가 찾아올 것입니다.

## 1. 깃허브에 프로필 작성하기
성장의 방향성을 잘 드러내기 위해 깃허브를 하나의 포트폴리오라고 생각하고 본인이 중점적으로 드러내고 싶은 부분을 정리해두면 좋습니다. 깃허브에는 자신의 프로필을 관리할 수 있는 기능이 있는데 이곳에서 자신을 간단히 소개할 수도 있고 내가 작업한 프로젝트 중 더 돋보이게 만들고 싶은 것을 상단으로 배치하여 개발자로서의 성장 과정과 앞으로의 가능성을 보여줄 수도 있습니다.

## 2. 링크드인에 프로필 작성하기
전문적인 자기 소개 페이지를 만들거나, 활동을 꾸준히 기록하는 것은 쉽지 않습니다. 특히 개발자로 시작하는 단계에서는 이런 부분이 부담스러울 수 있습니다. 이런 부담을 줄이기 위해 링크드인을 활용해보는 것을 추천합니다.

## 3. 블로그에 무언가를 시작하기 위한 준비 또는 진행 중인 과정을 공유하는 글 작성하기
많은 개발자가 블로그를 통해 본인의 경험을 기록합니다. 이 기록과 성장 과정을 보고 채용, 강연, 책 집필 등의 기회로 이어지기도 합니다. 깃허브에서는 코드 과정을 통해 하나의 기술적인 내용을 어떤 과정으로 만들었는지 알 수 있다면 블로그에서는 전반적인 개발 학습 과정과 개발자에게 필요한 역량을 어떤 과정으로 키우고 있는지 알 수 있기 때문입니다. 블로그에 나의 이야기를 적어보기 시작하는건 어떨까요?

**Comment by Seung-wan** [comment]
## 1. 깃허브에 프로필 작성하기
[깃허브 링크](https://github.com/Seung-wan)
매일 책을 꾸준히 읽으며 지식을 쌓고 있는 제 모습을 한 줄 소개로 표현해봤습니다!
상단에는 오픈 소스에 처음 PR을 날려본 것, NEXTSTEP 과정을 들었던 것, 리팩토링 챌린지를 했던것을 핀해놨는데요.
지금은 개인 프로젝트를 하지 않지만, 꾸준히 개발할 것을 하나 골라서 핀에 찝어 놓겠다는 목표를 세웠습니다!

## 2.
[링크드인 링크](https://www.linkedin.com/in/%EC%8A%B9%EC%99%84-%EC%9C%A0-a395b4212/)
사실 링크드인은 거의 신경을 쓰지 않았던 것 같아요.
계정만 만들고 관리를 하지 않았었는데, 간략하게라도 소개와 경력 사항을 작성해봤습니다.
커리어 내내 계속해서 내용을 추가하면 좋을 것 같네요!

## 3.
[블로그 링크](https://imnotadevleoper.tistory.com/entry/BFFBackend-For-Frontend%EB%8A%94-%EC%96%B4%EB%96%A4-%EB%AC%B8%EC%A0%9C%EB%A5%BC-%ED%95%B4%EA%B2%B0%ED%95%98%EB%82%98)
기술 블로그의 중요성에 대해 사실 이전부터 많이 듣기는 했었습니다.
그런데 귀찮다는... 이유로 쉽사리 작성하지 못했는데요.
최근에 글을 한번 작성해봤고 이제는 계속해서 이어나가보려고 합니다!

**Reply by nara04040** [reply]
글 잘보았습니다! 저 또한 링크드인과 블로그는 제대로 관리하지 않았는데.. 이번 기회에 저도 다시 관리해봐야겠습니다.
승완님께서는 기술 블로그를 작성하실 때 팁이 있을까요? 😀

**Reply by sgsg9447** [reply]
모두 팔로우 완료 !! ㅎㅎㅎ 블로그 글을 정말 잘쓰셨네요 종종 보러오겠습니다!

**Reply by Seung-wan** [reply]
@nara04040 
저는 먼저 공부하고 싶은 내가 모르는 토픽을 정해보는데요! 
어짜피 공부해야 할 거라면 공부도 하고 블로그에 기록하면서 스스로 정리도 하자!
그리고 주니어때는 블로그를 나름의 포트폴리오로도 사용하자! 라는 생각을 하고 있어요. 

최근에는 저는 요런 토픽을 정해봤어요!

1. 프론트엔드에서의 A/B 테스트에 대해 알아볼래요.
2. SSR 환경에서의 Suspense는 어떻게 동작할까요?

이렇게 토픽을 정하고 나서 구글에 검색을 하고, 공식 문서와 블로그를 포함해서 총 10개 정도의 탭을 열어보고 그 안에서 내용을 따로 노션에 정리하고 있어요. 정리하는 내용으로는 `이 기술은 어떤 문제를 해결하고 있지?`, `왜 이런 기술이 나왔지?`, `그래서 뭐가 좋은데?` 이런 질문에 답을 해보는 식으로 하고 있어요.

A/B 테스트를 예로 들면

- 그래서 A/B 테스트가 정확히 뭐지?
- 프론트에서 하긴 하는데.. 이걸 관리는 어디서 하는거지?
- A/B 테스트가 복잡해지면 관리를 어떻게 해야하지?
- A/B 테스트를 하면 얻을 수 있는 건 뭐지?
- A/B 테스트 말고 사용자의 행동을 파악하는 다른 테스트는 없나?
- 사용자가 별로 없을때는 의미있는 데이터보다 A/B 테스트를 세팅하는 비용이 더 크겠지?

이렇게 스스로 질문을 던지면서 이 생각의 흐름을 그대로 블로그에 써보면 재미도 있고 깊이 학습도 되서 좋은 것 같아요!

**Reply by Wu22e** [reply]
링크드인.. 저도 항상 준비해야지.. 하면서 미뤄왔는데, 승완님 쓰신거보고 저도 써야겠다고 다짐을 하게되네요.. 쓰게되면 공유 하겠습니다!

블로그도 종종 구경가겠습니다~

**Comment by logcat-io** [comment]
## 1. 깃허브에 프로필 작성하기
[GitHub](https://github.com/DevFrog92)
현재 프로젝트를 기획중이라서 도전하고 있는 과제를 핀해두었습니다.
지속적으로 학습하는 과정을 남기는 것을 목표로 하고 있습니다.

## 2. 링크드인에 프로필 작성하기
[Linkedin](https://www.linkedin.com/in/yongsu-kwon/)
유령 개정이었지만, 간단한 자기소개 부터 작성하였습니다.

## 3. 기술 블로그
[Blog](https://ditto-dev.tistory.com/)
블로그에 글을 작성하는 것이 매우 어려웠지만, 조금씩 글을 적기 위해서 노력하고 있습니다.

**Reply by AlbertImKr** [reply]
깔끔하게 잘 정리하셨어요. 잘보고 갑니다.
